src/cmd/root.go

```diff
@@ -88,7 +88,10 @@ func readConfig() {
 	}
 	viper.SetEnvPrefix("OPSLEVEL")
 	viper.AutomaticEnv()
-	viper.ReadInConfig()
+	err := viper.ReadInConfig()
+	if err != nil {
+		panic(err)
```

Comment:
This area of the code is called during cobra initialization so if this happens its OK to panic here